server/src/processing/TableDetectionModule/TableDetectionModule.ts

```diff
@@ -100,14 +95,15 @@ export class TableDetectionModule extends Module<Options> {
 	}
 
 	public main(doc: Document): Document {
-		const options: Options = { ...defaultOptions, ...this.options };
```

Comment:
Please check this!
I removed the Object.assign() as that kind of merge is no longer valid, and just used the this.options built in Module constructor

Reply:
👍